### DISCUSSION TITLE
Is it possible to support CucumberJS --format usage?

### DISCUSSION THREAD

**Discussion by jessica-jorgenson** [badeball/cypress-cucumber-preprocessor#1117] [Q&A] [answered]
Hello! Thanks for the awesome repo!

I wanted to ask if it was possible for the preprocessor to support formatting with the usage formatter, as documented here: https://github.com/cucumber/cucumber-js/blob/main/docs/formatters.md#usage

Or if it's not possible with how Cypress works.

Thanks!

**Comment by badeball** [comment, marked as answer]
See dry run: https://github.com/badeball/cypress-cucumber-preprocessor/blob/master/docs/diagnostics.md

**Reply by jessica-jorgenson** [reply]
Awesome, thank you!

Is the error `TypeError: Cypress.env is not a function` part of the 'especially experimental' issues, or something else entirely?

**Reply by badeball** [reply]
That sounds like something else entirely.
